Tests/ConfidenceProviderTests/ConfidenceFeatureProviderTest.swift

```diff
@@ -24,6 +25,41 @@ class ConfidenceFeatureProviderTest: XCTestCase {
         super.setUp()
     }
 
+    func testSlowFirstResolveWillbeCancelledOnSecondResolve() async throws {
+        class FakeClient: ConfidenceResolveClient {
+            var callCount = 0
+            var returnCount = 0
+
+            func resolve(ctx: ConfidenceStruct) async throws -> ResolvesResult {
+                callCount += 1
+                if callCount == 1 {
+                    // wait 2 seconds
```

Comment:
The wait below is for 1 sec
```suggestion
                    // wait 1 second
```